integration-tests/.load-test.mocharc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "exit": true,
+    "extension": ["ts"],
+    "parallel": false,
+    "require":  ["scaffolding/rootHooks.ts", "scaffolding/extrinsicHelpers.ts"],
+    "loader": "ts-node/esm",
+    "spec": ["./load-tests/**/*.test.ts"],
+    "timeout": 60_000,
```

Comment:
Does json format supports numbers with underscore? Seems Github viewer doesn't seem to like it.

Reply:
I think it works due to being processed by typescript

Reply:
I'll remove it in a future PR